jest.setup.js

```diff
@@ -23,3 +23,11 @@ jest.mock(
 		getHostAppNamespace: jest.fn(),
 	} )
 );
+
+// The module `yjs` throws an error if it's imported more than once.
+// This module is only used in the `@wordpress/sync` package which is not used
+// by the native version. Due to the nature of some of the tests and the need to
+// re-importing modules, we have to explicitly mock the module to avoid disruptions
```

Comment:
Why/How is it imported multiple times? Why is this only an issue in the testing environment?

Reply:
This error is happening in the following test files:
- https://github.com/wordpress-mobile/gutenberg-mobile/blob/trunk/src/test/jetpack-editor-setup.js
- https://github.com/wordpress-mobile/gutenberg-mobile/blob/trunk/src/test/index.js

As far as I checked, seems it's being imported multiple times when [mocking the setup logic](https://github.com/wordpress-mobile/gutenberg-mobile/blob/1874cb1e0e154405f33f7ef726e7434ce640e045/src/test/index.js#L17) and when [isolating modules](https://github.com/wordpress-mobile/gutenberg-mobile/blob/1874cb1e0e154405f33f7ef726e7434ce640e045/src/test/jetpack-editor-setup.js#L59-L62). Other tests are not flagging this, so I assume this error is specific to the test environment. In any case, it's not a bad idea to test the demo app just in case.

As a side note, the `yjs` module has been incorporated in https://github.com/WordPress/gutenberg/commit/79dbfc57a0f3918a299f5856f69561f695ff8896#diff-b8c5399ce4ea3561ee97ec17d5ef1a45283f458d5c952647710170cb5008dfd3R33.

Reply:
I see. To state the obvious: it would be ideal to identify the root issue and resolve it in a manner that does not require a cryptic mock, e.g. refactoring the module loading approach. However, I do not necessarily think that is worth the effort currently. I think this mocking approach is sufficient with the inline comment.

Reply:
I totally agree, getting to the root cause of the issue is important to discard potential side effects in the editor. I'll create a GitHub issue as a follow-up. Note that no other tests failed with this issue, this makes me think that is likely that it's simply caused by specific test cases, i.e. not a real case that could be reproduced in the editor.

Reply:
I created https://github.com/wordpress-mobile/gutenberg-mobile/issues/6055 as a follow-up on this.